test/notification_test.html

```diff
@@ -3,10 +3,13 @@
 <head lang="en">
     <meta charset="UTF-8">
     <title></title>
+   <script src="packages/test/dart.js"></script>
+   <link rel="x-dart-test" href="notification_test.dart">
+<!--   <script src="notification_test.dart" type="application/dart"></script>
+   <script src="packages/browser/dart.js"></script> -->
+
 </head>
 <body>
-<button id="b">Click me</button>
-<script src="notification_test.dart" type="application/dart"></script>
-<script src="packages/browser/dart.js"></script>
+    <button id="b">Click me</button>
```

Comment:
This button is not used in your test. So I would say `test/notification_test.html` can be removed.